internal/pkg/cli/job_deploy_test.go

```diff
@@ -109,3 +109,81 @@ func TestJobDeployOpts_Validate(t *testing.T) {
 		})
 	}
 }
+
+func TestJobDeployOpts_Ask(t *testing.T) {
+	testCases := map[string]struct {
+		inAppName  string
+		inEnvName  string
+		inJobName  string
+		inImageTag string
+
+		wantedCalls func(m *mocks.MockwsSelector)
+
+		wantedJobName  string
+		wantedEnvName  string
+		wantedImageTag string
+		wantedError    error
+	}{
+		"prompts for environment name and job names": {
```

Comment:
should we also add a testcase for the image tag or bake in into this test case?

Reply:
per offline conversation, will add a testcase for image tag (here and in svc_deploy) later, after the `getVersionTag` method is refactored (https://github.com/aws/copilot-cli/issues/1413).